aspnetcore/tutorials/index.md

```diff
@@ -16,18 +16,18 @@ ms.prod: asp.net-core
 The following step-by-step guides for developing ASP.NET Core applications are available:
 
 ## Building web applications
-* [Introduction to ASP.NET Core MVC on Mac or Linux ](first-mvc-app-xplat/index.md)
+* [Introduction to ASP.NET Core MVC on macOS, Linux or Windows](first-mvc-app-xplat/index.md)
 * [Introduction to ASP.NET Core MVC with Visual Studio](first-mvc-app/index.md)
 * [Getting started with ASP.NET Core and Entity Framework Core using Visual Studio](../data/ef-mvc/index.md)
-* [Your first ASP.NET Core application on a Mac or Linux using Visual Studio Code](your-first-mac-aspnet.md)
+* [Your first ASP.NET Core application on a macOS or Linux using Visual Studio Code](your-first-mac-aspnet.md)
```

Comment:
on ~~a~~ macOS

This is now very similar to xplat MVC - can we provide guidance on which to choose somehow?  Should they do this one first, so we would put it first in the list?  Or just if they're short on time?  What does it give that they can't get in xplat MVC? Should we retire this one?